Makefile

```diff
@@ -65,8 +65,8 @@ test-go: $(COG_EMBEDDED_WHEEL)
 	$(GO) run gotest.tools/gotestsum -- -timeout 1200s -parallel 5 ./... $(ARGS)
 
 .PHONY: test-integration
-test-integration: $(COG_BINARIES)
-	PATH="$(PWD):$(PATH)" $(TOX) -e integration
+test-integration: $(COG_BINARIES) $(COG_WHEEL)
+	PATH="$(PWD):$(PATH)" $(TOX) run --installpkg $(COG_WHEEL) -e integration
```

Comment:
Hmmm, I don't think we should be installing the built cog into the tox environment here. The integration tests are supposed to be testing `cog` the binary, not the python package.

Reply:
I know _why_ you've done this... I just don't think it's a particularly pleasant solution to the problem.

A couple of alternatives to consider:

- compare the output of `cog --version` and `python -c "import cog; print(cog.__version__)"` from inside the built image -- they will not be identical except for release builds, so some string futzing (specifically, removing the local scheme parts of the PEP440 version) will be needed
- allow `cog --version` to report both the cog version and the version of the embedded wheel, which is contained in the file name and can be reliably parsed out

Reply:
What would you suggest to check against the cog version?